src/users.service.ts

```diff
@@ -0,0 +1,31 @@
+import { IUser } from "./user.interface";
+
+export class UserService {
+  private users: IUser[] = [
+    { name: "dietl", amount: 20 },
+    { name: "laciar", amount: 60 },
+  ];
+
+  public initial() {
```

Comment:
In my branch at the file src/users.service.ts  between lines 6 and 9 I have the following code:

` public initial() {
    return "HELLO WORLD PLEASE SEND USERS";
  }
  public save(user: IUser) {`

Where can the problem be?
Thanks

Reply:
remove this, from my point of view, is code not used and useless.
otherwise, please explain it.


```public initial() { return "HELLO WORLD PLEASE SEND USERS"; } ```

Reply:
done thanks!